.github/workflows/update_buildpack.yml

```diff
@@ -0,0 +1,58 @@
+on:
+  push:
+    branches: ["main"]
+  pull_request:
+    types: [opened, edited, synchronize, reopened]
+jobs:
+  Test_Result:
+    runs-on: ubuntu-latest
+    steps:
+
+      
+  
+
+      - name: Remove Older Build pack
+        env:
+         HEROKU_TOKEN: ${{ secrets.HEROKU_API_KEY }}
+        run: |
+          echo "REMOVE_OLDER_BUILDPACK=$(curl  -o /dev/null -s -w "%{http_code}\n" -n -X PUT https://api.heroku.com/apps/650b637e-ffd2-45c5-bd24-11e7e49487d2/buildpack-installations -H "Content-Type: application/json" -H "Accept: application/vnd.heroku+json; version=3" -H  "Authorization: $HEROKU_TOKEN" -d '{"updates":[]}')" >> $GITHUB_ENV
+   
+          
+      - name: Add a newer Build Pack of current branch
+        env:
+         HEROKU_TOKEN: ${{ secrets.HEROKU_API_KEY }}
+        run: |
+          echo "UPDATE_OLDER_BUILDPACK=$(curl  -o /dev/null -s -w "%{http_code}\n" -n -X PUT https://api.heroku.com/apps/650b637e-ffd2-45c5-bd24-11e7e49487d2/buildpack-installations -H "Content-Type: application/json" -H "Accept: application/vnd.heroku+json; version=3" -H  "Authorization: $HEROKU_TOKEN" -d "{\"updates\":[{ \"buildpack\": "\"https://github.com/SalesforceDocs-Bot/sfdocs-heroku-buildpack-nodejs#${GITHUB_HEAD_REF}\""}]}")" >> $GITHUB_ENV
```

Comment:
This URL should be available in the payload using this variable repo_url so no need to hardcode